mistakedetection.tests/src/ca/mcgill/sel/mistakedetection/tests/MistakeDetectionPerformanceAnalysis.java

```diff
@@ -264,6 +342,298 @@ public void testStudentSolution20() {
+
+    var comparison = MistakeDetection.compare(instructorSolution, studentSolution, true);
+     
+
+  }
+
+  @Test
+  public void testStudentSolution23() throws Exception {
+    var name = "G13_5";
+    var studentClassDiagram = getStudentClassDiagram(name);
+    var studentSolution = studentSolutionFromClassDiagram(studentClassDiagram);
+
+    var comparison = MistakeDetection.compare(instructorSolution, studentSolution, true);
+     
+
+  }
+
+  @Test
+  public void testStudentSolution24() throws Exception {
+    var name = "G13_9";
+    var studentClassDiagram = getStudentClassDiagram(name);
+    var studentSolution = studentSolutionFromClassDiagram(studentClassDiagram);
+
+    var comparison = MistakeDetection.compare(instructorSolution, studentSolution, true);
+     
+
+  }
+
+  @Test
+  public void testStudentSolution25() throws Exception {
+    var name = "G13_15";
+    var studentClassDiagram = getStudentClassDiagram(name);
+    var studentSolution = studentSolutionFromClassDiagram(studentClassDiagram);
+
+    var comparison = MistakeDetection.compare(instructorSolution, studentSolution, true);
+     
+
+  }
+
+  @Test
+  public void testStudentSolution26() throws Exception {
+    var name = "G14_2";
+    var studentClassDiagram = getStudentClassDiagram(name);
+    var studentSolution = studentSolutionFromClassDiagram(studentClassDiagram);
+
+    var comparison = MistakeDetection.compare(instructorSolution, studentSolution, true);
+     
+
+  }
+
+  @Test
+  public void testStudentSolution27() throws Exception {
+    var name = "G14_7";
+    var studentClassDiagram = getStudentClassDiagram(name);
+    var studentSolution = studentSolutionFromClassDiagram(studentClassDiagram);
+
+    var comparison = MistakeDetection.compare(instructorSolution, studentSolution, true);
+     
+
+  }
+
+  @Test
+  public void testStudentSolution28() throws Exception {
+    var name = "G14_12";
+    var studentClassDiagram = getStudentClassDiagram(name);
+    var studentSolution = studentSolutionFromClassDiagram(studentClassDiagram);
+
+    var comparison = MistakeDetection.compare(instructorSolution, studentSolution, true);
+     
+  }
+
+  @Test
+  public void testStudentSolution29() throws Exception {
+    var name = "G15_5";
+    var studentClassDiagram = getStudentClassDiagram(name);
+    var studentSolution = studentSolutionFromClassDiagram(studentClassDiagram);
+
+    var comparison = MistakeDetection.compare(instructorSolution, studentSolution, true);
+     
+  }
+
+  @Test
+  public void testStudentSolution30() throws Exception {
+    var name = "G15_9";
+    var studentClassDiagram = getStudentClassDiagram(name);
+    var studentSolution = studentSolutionFromClassDiagram(studentClassDiagram);
+
+    var comparison = MistakeDetection.compare(instructorSolution, studentSolution, true);
+     
+  }
+
+  public static void produceExcelSheet(Comparison comparison, String name, String location) throws Exception {
```

Comment:
This method is quite long and complex so it should be broken up into smaller pieces that are easier to understand and maintain